ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/ControlAnnotationSublayerVisibility/ControlAnnotationSublayerVisibility.cpp

```diff
@@ -55,7 +55,7 @@ QString defaultDataPath()
 } // namespace
 
 // sample MMPK location
-const QString sampleFileAnno {"/ArcGIS/Runtime/Data/mmpk/GasDeviceAnno.mmpk"};
+const QString sampleFileAnno {"/ArcGIS/Runtime/Data/mmpk/GasDeviceAnno.mmpk"}; // why not in namespace?
```

Comment:
I don't know if there is a reason why this variable is outside anonyme namespace.